build/linux/installer/conf/fluent-bit-network-flow-logs.conf

```diff
@@ -0,0 +1,30 @@
+[INPUT]
+    Name                tail
+    Alias               network_flow_input
+    Tag                 oms.networkflow.*
+    Path                /var/run/acns/hubble/events.log
```

Comment:
since path changed, after the RP changes rolled out and we would need to update path to /var/log/acns/hubble/events.log.